src/app/my-messages/my-messages.component.spec.ts

```diff
@@ -27,12 +38,39 @@ const expectedNotifications: any[] = [
     registration: {
       id: 'b9f7a830-6437-4585-980c-b6820b6f03fb',
       user: {
-        id: '251c2a3b-b737-4622-8060-196d5e297ebc',
-        userName: 'testbed',
-        firstName: 'Tabia',
-        lastName: 'Testbed',
-        email: 'tabia.testbed@myskills.io',
-        coach: false,
+        id: authenticatedUser.userId,
+        userName: 'tester',
+        firstName: 'Toni',
+        lastName: 'Tester',
+        email: 'toni.tester@myskills.io',
+        coach: false
+      },
+      approvedByUser: null,
+      approvedByCommunity: true,
+      community: {
+        id: '22c1ad17-4044-45a7-940c-22f1beeb7992',
+        title: 'Some closed community',
+        type: 'CLOSED',
+        description: 'Some closed community description',
+        links: [],
+        managers: [],
+        skills: []
+      }
+    }
+  }),
+  Util.createNotificationInstance({
+    type: NotificationType.REQUEST_TO_JOIN_COMMUNITY,
+    id: 'e84cacac-4fba-4cea-b7e6-7e3d9841b0a6',
+    creationDatetime: new Date(),
+    registration: {
+      id: 'b9f7a830-6437-4585-980c-b6820b6f03fb',
+      user: {
+        id: authenticatedUser.userId,
+        userName: 'tester',
+        firstName: 'Toni',
+        lastName: 'Tester',
+        email: 'toni.tester@myskills.io',
+        coach: false
       },
       approvedByUser: null,
```

Comment:
If it is a request to join a community, I think the field `approvedByUser` should be set to `true` while `approvedByCommunity` should be set to `null`.

Reply:
thank you. fixed